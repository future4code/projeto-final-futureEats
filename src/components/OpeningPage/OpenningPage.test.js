import React from "react"
import { render } from "enzyme"
import {OpeningPage} from "./OpeningPage"

describe("OpeningPage", () => {
    it("OpeningPage Snapshot", () => {
        const tree = render(<OpeningPage />)
        expect(tree).toMatchSnapshot()
    })
})