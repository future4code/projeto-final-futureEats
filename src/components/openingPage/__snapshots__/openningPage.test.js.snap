// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OpeningPage OpeningPage Snapshot 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "sc-bdVaJa dIkVTY",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "sc-bwzfXH cYucvQ",
          "src": "logo-future-eats.png",
        },
        "children": Array [],
        "name": "img",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
          "src": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
          "src": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
